frontend/src/app/testQueue/AoEForm/PriorTestInputs.tsx

```diff
@@ -161,8 +170,10 @@ const PriorTestInputs: React.FC<Props> = ({
               // Fill in last test info using this data
               // TODO: update when test history has test type
               setPriorTestType("2");
-              setPriorTestDate((lastTest.dateTested || "").split("T")[0]);
-              setPriorTestResult(lastTest?.result);
+              setPriorTestDate(
+                (lastTest.dateTested || "").split("T")[0] as FormattedDate
```

Comment:
I think a better approach here would be to type `lastTest.dateTested` as an ISODate and do the formating when we get the value for `lastTest.dateTested`

Reply:
Take a look at 7026199 and let me know if that's what you had in mind.

Reply:
Actually that previous commit was subtly wrong - the AoE form is also included in patient experience when conducting a test, so that formatting needs to happen in the `AoEForm` component rather than `AoEModalForm`. See 90b51cb.

Shoutout to the end-to-end tests for catching this.